### ISSUE TITLE
Integrate firebase realtime database example

### ISSUE THREAD

**Issue by nahuelhds** [jthegedus/firebase-functions-next-example#15]
Hi @jthegedus . This is not an issue but a question. Are you interested if I create a PR for adding an example page doing a firebase realtime database example?

I ask this because I've been several days working on this until I've found the way to do it properly. I dig on the web, and of course I had a learning curve that cost me that amount of time.

But in the end I think that it'd be nice for people to have both sides of firebase in the same example: cloud functions for the backend tasks and frontend functions for the realtime database API.

Basically, I took the way that is done at https://github.com/now-examples/next-news with minor modifications.

Let me know and I'll do the PR between today and tomorrow.

Regards and nice work, man. It helped me a lot

**Comment by jthegedus** [comment]
@nahuelhds Hey mate, sorry was on vacation. I would happily accept a PR for that example. I would, however, prefer it to be a separate example to this one as I have been working towards an single repo of all examples that gradually adds different aspects of a stack for people to follow the progression easily. That repo is https://github.com/jthegedus/blog-code , so if you could PR that example there, it would be greatly appreciated.

I am intending on moving all my examples there, including this one, however, since this one is so popular I was going to archive it only after the new repo reaches equal popularity.

Also, as I stated, I have been working towards an single example repo and intend to write about all of the entries. When I write about each example I will of course credit all contributors to each example, so please be sure you're okay with that arrangement before contributing (I will add this disclaimer to that repo).

Thanks for reaching out and the kind words :smiley:

**Comment by nahuelhds** [comment]
You know I finally never could to make FB Database to work when going live. It worked when testing locally with `yarn serve` but when I've uploaded the files to FB Hosting, I just got error in the very moment when trying to call to firebase.

So I finally ended up working with https://zeit.co/now

No complicated stuff, just work with nextjs and deploy easily. Anyway, I'll try to upload in the near future the project to Firebase hosting. If I make it, I'll let you know if you're interested.

Your work helped me to learn a lot. I'm grateful for that.

Regards!
